cmd/incusd/networks.go

```
@@ -489,6 +489,19 @@ func networksPost(d *Daemon, r *http.Request) response.Response {
 			return response.SmartError(err)
 		}
 
+		n, err := network.LoadByName(s, projectName, req.Name)
+		if err != nil {
+			return response.SmartError(fmt.Errorf("Failed loading network: %w", err))
+		}
+
+		err = s.Authorizer.AddNetwork(r.Context(), projectName, req.Name)
+		if err != nil {
+			logger.Error("Failed to add network to authorizer", logger.Ctx{"name": req.Name, "project": projectName, "error": err})
```

So you don't want to return an error in this case?